bin/lib/Logos/Generator/Base/Property.pm

```diff
@@ -60,7 +60,7 @@ sub definition {
 			$setter_func .= "{ objc_setAssociatedObject(self, (void *)$propertyGetterName, rawValue, $propertyAssociationPolicy); }";
 		}
 
-		return "$getter_func;\n$setter_func";
+		return "$getter_func\n$setter_func";
```

Comment:
Actually the newline is the character that should be changed. I now notice this was introduced in the previous commit.
```suggestion
		return "$getter_func; $setter_func";
```